src/go/rpk/pkg/cli/cmd/container/status_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package container
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/cli/cmd/container/common"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatus(t *testing.T) {
+	tests := []struct {
+		name           string
+		client         func() (common.Client, error)
+		expectedErrMsg string
+		expectedOutput []string
```

Comment:
[nit]: for consistency with the new tests, feel free to ignore.

```suggestion
		expErrMsg string
		exp       []string
```

Reply:
Agree with the nit, though it looks like this is taken from the existing tests -- we can fix this in a whole-package refactoring.